src/Routes/Devices/Inventory.js

```diff
@@ -160,6 +162,10 @@ const Inventory = ({ historyProp, locationProp, showHeaderProp }) => {
     setIsDeleteModalOpen(false);
   }
 
+  useEffect(() => {
+    chrome?.updateDocumentTitle?.('Systems - Inventory | Edge management');
```

Comment:
@acruzgon could you confirm if it's `Edge management` or `Edge Management`?

Reply:
Documentation shows `Edge management` but there is some places the second word gets uppercase `M`